packages/expo-constants/ios/EXConstants/EXConstantsService.m

```diff
@@ -284,7 +285,7 @@ + (NSString *)deviceModel
       deviceModel = @"Apple TV";
     }
   }
-  return deviceModel;
+  return deviceModel ?: @"Unsupported";
```

Comment:
nil seems like a better value than Unsupported here -- callers need to handle it anyway